src/components/VideoList/VideoList.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import VideoItem from './VideoItem';
+
+const VideoListStyled = styled.div`
+  padding: 20px;
+  margin: 0px auto;
+  flex: 1 1 0%;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  box-sizing: border-box;
+`;
+
+const VideoList = (props) => {
+  const displayItems = (items) => {
+    let newItems = [];
+    if (items) {
+      newItems = items.map((item) => (
+        <VideoItem
+          key={item.etag}
+          title={item.snippet.title}
+          description={item.snippet.description}
+          image={item.snippet.thumbnails.high.url}
+        />
+      ));
+    }
+    return newItems;
+  };
```

Comment:
This is ok, but you can remove the need of checking for truthiness of `items` by declaring a default value to the function parameter like:

```jsx
  const displayItems = (items = []) = {
    return items.map(/* map code */);
  }
```
and actually the check might not be needed since your checking the length of items before calling the function in line `35`, you can move the map there like

```jsx
     {props.collection.items.map(/* map items */)}
```